core/examples/generated/lib/railjson_generator/physical_approximations.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from typing import List
+
+import json
+from railjson_generator.schema.infra.range_elements import TrackRange
+
+from schemas import physical_approximations
+
+
+@dataclass
+class ElectricalProfileTest:
+    value: str
+    power_class: str
+    track_ranges: List[TrackRange] = field(default_factory=list)
+
+    def add_track_range(self, track, begin, end):
+        self.track_ranges.append(TrackRange(track=track, begin=begin, end=end))
+
+    def to_rjs(self):
+        return physical_approximations.ElectricalProfile(
+            value=self.value,
+            power_class=self.power_class,
+            track_ranges=[track_range.to_rjs() for track_range in self.track_ranges],
+        )
+
+
+@dataclass
+class PhysicalApproximations:
```

Comment:
I know it's not the only place where it's named this way, but why is this class called `PhysicalApproximations`? I don't think it's a helpful name, a bunch of things are physical approximations

Reply:
This is the best I could manage sadly, by physical approximations, I meant more "discutable approximations", approximations which will be treated as optional...

It felt weird to have files and directories named "electrical_profiles", since electrical profiles are akin to speed sections...

What name would you have given this ? (If I change the name now, I would have to change api, osrd-dags and core)

Reply:
how about `ExternalInputs`?

Reply:
`ExternalGeneratedInputs`